<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-03-07T11:08:53Z" name="Default suite" finished-at="2024-03-07T11:09:21Z" duration-ms="27459">
    <groups>
    </groups>
    <test started-at="2024-03-07T11:08:53Z" name="Default test" finished-at="2024-03-07T11:09:21Z" duration-ms="27459">
      <class name="com.adactinproject.testcases.NewUserRegistrationPageTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.adactinproject.testcases.NewUserRegistrationPageTest@183ec003]" started-at="2024-03-07T11:08:53Z" name="setup" finished-at="2024-03-07T11:08:59Z" duration-ms="5833" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="NewUserRegistrationPageTitleTest()[pri:0, instance:com.adactinproject.testcases.NewUserRegistrationPageTest@183ec003]" started-at="2024-03-07T11:08:59Z" name="NewUserRegistrationPageTitleTest" finished-at="2024-03-07T11:08:59Z" duration-ms="43" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- NewUserRegistrationPageTitleTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.adactinproject.testcases.NewUserRegistrationPageTest@183ec003]" started-at="2024-03-07T11:08:59Z" name="tearDown" finished-at="2024-03-07T11:09:00Z" duration-ms="380" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.adactinproject.testcases.NewUserRegistrationPageTest@183ec003]" started-at="2024-03-07T11:09:00Z" name="setup" finished-at="2024-03-07T11:09:03Z" duration-ms="3212" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="enterUserDetails()[pri:0, instance:com.adactinproject.testcases.NewUserRegistrationPageTest@183ec003]" started-at="2024-03-07T11:09:03Z" name="enterUserDetails" finished-at="2024-03-07T11:09:16Z" duration-ms="13576" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- enterUserDetails -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.adactinproject.testcases.NewUserRegistrationPageTest@183ec003]" started-at="2024-03-07T11:09:16Z" name="tearDown" finished-at="2024-03-07T11:09:17Z" duration-ms="539" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.adactinproject.testcases.NewUserRegistrationPageTest@183ec003]" started-at="2024-03-07T11:09:17Z" name="setup" finished-at="2024-03-07T11:09:20Z" duration-ms="3357" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="logoImageTest()[pri:0, instance:com.adactinproject.testcases.NewUserRegistrationPageTest@183ec003]" started-at="2024-03-07T11:09:20Z" name="logoImageTest" finished-at="2024-03-07T11:09:20Z" duration-ms="89" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertFalse(Assert.java:65)
at org.testng.Assert.assertFalse(Assert.java:75)
at com.adactinproject.testcases.NewUserRegistrationPageTest.logoImageTest(NewUserRegistrationPageTest.java:36)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoImageTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.adactinproject.testcases.NewUserRegistrationPageTest@183ec003]" started-at="2024-03-07T11:09:20Z" name="tearDown" finished-at="2024-03-07T11:09:21Z" duration-ms="380" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.adactinproject.testcases.NewUserRegistrationPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
